### DISCUSSION TITLE
建议针对每堂课的内容附加参考资料和扩展学习建议

### DISCUSSION THREAD

**Discussion by hbsun2113** [mlc-ai/mlc-zh#108] [Ideas]
除了老师在视频中所讲的内容外，希望能更全面、深入了解相关知识。是否可以针对每堂课，补充些参考资料和提供扩展学习的建议？

**Comment by hbsun2113** [comment]
特别是第7堂课，没有GPU编程基础，希望能提供些补充学习资料。

**Reply by yzh119** [reply]
粗略学习可以参考这门课程 https://tschmidt23.github.io/cse599i/ 
更详细的内容参阅官方文档即可 https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html
